README.md

```diff
@@ -27,14 +27,18 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Be prepared to demonstrate your understanding of this week's concepts by answering questions on the following topics. You might prepare by writing down your own answers before hand.
 
 1. What is the DOM?
+   the DOM is contains a webpage when it is built, and has different nodes that have built in methods that manipulate the html.
```

Comment:
The DOM is an object representation of the HTML on the page. The DOM can be manipulated using Javascript